legend-engine-xts-dataquality/legend-engine-xt-dataquality-pure/src/main/resources/core_dataquality/generation/dataquality.pure

```diff
@@ -534,3 +545,28 @@ function meta::external::dataquality::buildRelationFilterExpression(dqRelationVa
    )->evaluateAndDeactivate();
 }
 
+function meta::external::dataquality::buildLimitFilterExpression(f: FunctionExpression[1], limit: Integer[1], returnType:GenericType[1]):FunctionExpression[1]
+{
+  ^SimpleFunctionExpression(func=take_T_MANY__Integer_1__T_MANY_,
+                            parametersValues=[$f, ^InstanceValue(values=$limit->toOne(), genericType=^GenericType(rawType=Integer), multiplicity=PureOne)],
+                            functionName=take_T_MANY__Integer_1__T_MANY_.name,
+                            genericType=$returnType,
+                            multiplicity = ZeroMany,
+                            importGroup=system::imports::coreImport)->evaluateAndDeactivate();
+}
+
+function meta::external::dataquality::buildFromRuntimeExpression(f: FunctionExpression[1], runtime: Runtime[1], returnType:GenericType[1]):FunctionExpression[1]
+{
+  ^SimpleFunctionExpression(func=from_T_m__Runtime_1__T_m_,
+                            functionName=from_T_m__Runtime_1__T_m_.name,
+                            importGroup=system::imports::coreImport,
+                            genericType=$returnType,
+                            multiplicity=ZeroMany,
```

Comment:
lets double check that ZeroMany is correct here. I would expect multiplicity to be the same as the paramValue given the function signature